tests/foreman/api/test_permission.py

```diff
@@ -41,17 +63,17 @@ def test_search_by_name(self, permission_name):
         is the one searched for.
 
         """
+        permission_name = random.choice(self.permission_names)
         if (
                 permission_name == 'power_compute_resources_vms' and
                 bz_bug_is_open(1198731)):
-            self.skipTest('BZ 1198731 is open.')
+            permission_name = random.choice(self.permission_names)
         result = entities.Permission(name=permission_name).search()
         self.assertEqual(len(result), 1, permission_name)
         self.assertEqual(permission_name, result[0]['name'])
 
     @run_only_on('sat')
-    @ddt_data(*PERMISSION_RESOURCE_TYPES)
```

Comment:
This will decrease the number of tests, and we can't check for server software in the import time.